docs/user-guide/garbage-collection.md

```diff
@@ -0,0 +1,84 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/user-guide/garbage-collection.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Garbage Collection
+
+Garbage collection is managed by kubelet automatically, mainly including unreferenced
+images and dead containers. kubelet applies container garbage collection every minute
+and image garbage collection every 5 minutes.
+
+### Image Collection
+
+kubernetes manages lifecycle of all images through `imageManager`。
+The policy for garbage collecting images we apply takes two factors into consideration,
+`HighThresholdPercent` and `LowThresholdPercent`. Disk usage above the the high threshold
+will trigger garbage collection,  and never down to low threshold.
+
+### Container Collection
+
+The policy for garbage collecting containers we apply takes on three variables, which can
+be user-defined. See also [here](#User Configuration).
+kubelet sorts out containers which are unidentified or stay out of bounds set by previous
+mentioned three flags. For the latter case, the oldest containers are removed first.
+When kubelet cleans up the dead containers, the associated volumes will be taken care of as well.
+
+> Note that we will skip the containers that are not managed by kubelet.
+
+### User Configuration
+
+Users are free to set their own value to address image garbage collection.
+
+- `image-gc-high-threshold`, the percent of disk usage after which image garbage collection is
+always run. Default is 90%
+- `image-gc-low-threshold`, the percent of disk usage before which image garbage collection is
```

Comment:
> the percent of disk usage before which image garbage collection is never run

This reads a bit awkward to me.  Is this used for anything other than determining how much space to attempt to free up?  If not, maybe just say something like "garbage collection will attempt to free space until this percent of disk is free"